test/AcceleraterTest.cpp

```
@@ -0,0 +1,118 @@
+	accel = mockSensor.getAcceleration();
+	moving = mockSensor.isMoving();
+
+	std::cout << "  Acceleration: X=" << accel[0] << ", Y=" << accel[1] << ", Z=" << accel[2]
+						<< std::endl;
+	std::cout << "  Moving: " << (moving ? "YES" : "NO") << std::endl;
+
+	std::cout << "Test completed." << std::endl;
+}
+
+// Entry point for test file
+int runTests() {
```

Cppcheck (2.10-2)

[style] The function 'runTests' is never used.

(unusedFunction)